app/code/Magento/NewRelicReporting/Plugin/StatPlugin.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\NewRelicReporting\Plugin;
+
+use Magento\Framework\Profiler\Driver\Standard\Stat;
+use Magento\NewRelicReporting\Model\Config;
+use Magento\NewRelicReporting\Model\NewRelicWrapper;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class StatPlugin handles single Cron Jobs transaction names
+ */
+class StatPlugin
+{
+    const TIMER_NAME_CRON_PREFIX = 'job';
```

Comment:
Make const private

Reply:
Does it mean that I can throw this PR to the trash as long as we don't have 2.4-dev branch?